pkg/netbox/api/prefix.go

```diff
@@ -98,9 +100,17 @@ func (r *NetboxClient) UpdatePrefix(prefixId int64, prefix *netboxModels.Writabl
 }
 
 func (r *NetboxClient) DeletePrefix(prefixId int64) error {
+	// assuming id starts from 1 when Prefix is created so 0 means that Prefix doesn't exist
+	if prefixId == 0 {
```

Comment:
Not sure what this check is about. Is it to catch the case where the prefix was never created in NetBox?